rosbot_xl_gazebo/config/gz_stereolabs_zed_remappings.yaml

```diff
@@ -0,0 +1,22 @@
+- topic_name: <robot_namespace><zed>/zed_node/rgb/camera_info
+  ros_type_name: sensor_msgs/msg/CameraInfo
+  gz_type_name: ignition.msgs.CameraInfo
+
+- topic_name: <robot_namespace><zed>/zed_node/rgb/image_rect_color
+  ros_type_name: sensor_msgs/msg/Image
+  gz_type_name: ignition.msgs.Image
+
+- ros_topic_name: <robot_namespace><zed>/zed_node/depth/camera_info
+  gz_topic_name: <robot_namespace><zed>/zed_node/camera_info
+  ros_type_name: sensor_msgs/msg/CameraInfo
+  gz_type_name: ignition.msgs.CameraInfo
+
+- ros_topic_name: <robot_namespace><zed>/zed_node/depth/depth_registered
+  gz_topic_name: <robot_namespace><zed>/zed_node/depth
+  ros_type_name: sensor_msgs/msg/Image
+  gz_type_name: ignition.msgs.Image
+
+- ros_topic_name: <robot_namespace><zed>/zed_node/point_cloud/cloud_registered
+  gz_topic_name: <robot_namespace><zed>/zed_node/depth/points
+  ros_type_name: sensor_msgs/msg/PointCloud2
+  gz_type_name: ignition.msgs.PointCloudPacked
```

Comment:
With the newest zed_ros2_wraper the topic is the same for all camera. Need change also in simulation.py
![image](https://github.com/husarion/rosbot_xl_ros/assets/126687345/f6284e1a-e921-464a-9435-8bdec64576d7)

Reply:
Great news!